src/test/scala/eventstore/StreamSourceSpec.scala

```diff
@@ -290,30 +291,6 @@ class StreamSourceSpec extends SourceSpec {
       override def eventNumber = Some(EventNumber(0))
     }
 
-    "resubscribe from same position" in new SourceScope {
```

Comment:
what is a reason for deleting these ?

Reply:
It seems to originate from `StreamSubscriptionActorSpec` where the subscription actor sent "LiveProcessingStarted" message when switching to live mode, and where it was possible to test that it didn't send it twice.

When we can't tell when we go into live mode, it is not possible to assert that we not go live twice.

Reply:
If you want, I can add it again in this form:

```scala
    "handle unexpected resubscribe from same position while subscribed" in new SourceScope {
      connection expectMsg readEvents(0)
      connection reply readCompleted(1, endOfStream = true, event0)
      connection expectMsg subscribeTo
      connection reply subscribeCompleted(1)
      connection reply subscribeCompleted(1)
      expectNoActivity()

      override def eventNumber = Some(EventNumber(0))
    }
```